### PR TITLE
api: add feedback api for interactions

### PR DESCRIPTION
**Pull request by lucklove**
## Related issue number

Closes https://github.com/pingcap/LinguFlow/issues/56

## Checks

- [x] I've included any doc changes needed for https://github.com/pingcap/LinguFlow/.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: api/application.py [hunk 1]
```diff
@@ -285,6 +289,52 @@ def get_interaction(self, interaction_id: str) -> InteractionInfoResponse:
...
+                success=False,
+                message=f"Langfuse was not configured correctly application {interaction.app_id}.",
+            )
+
+        # create score
+        langfuse = Langfuse(
+            public_key=app.langfuse_public_key,
+            secret_key=app.langfuse_secret_key,
+        )
+        langfuse.score(
```
Comment (freedeaths) [inline api/application.py:328]: It returns a StatefulClient. It seems that line 336 runs whenever langfuse.score is successful or failed.

**Review by freedeaths — CHANGES_REQUESTED** [review]

File: api/application.py (hunk 1 above)
Comment (lucklove) [reply api/application.py:328]: updated

**Review by freedeaths — APPROVED** [review]
Exactly langfuse didn't provide any specified exception so that we can do nothing now although it's a little weird to return success always.

cc @lilyjazz

**Review by lilyjazz — APPROVED** [review]
LGTM

**Pull request merged**